recipes-graphics/userland/userland.inc

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "This repository contains the source code for the ARM side \
+libraries used on Raspberry Pi. These typically are installed in /opt/vc/lib \
+and includes source for the ARM side code to interface to: EGL, mmal, GLESv2,\
+vcos, openmaxil, vchiq_arm, bcm_host, WFC, OpenVG."
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENCE;md5=0448d6488ef8cc380632b1569ee6d196"
+
+COMPATIBLE_MACHINE = "^rpi$"
+
+SRCBRANCH = "master"
+SRCFORK = "raspberrypi"
+SRCREV = "188d3bfe4a0ac36b119a2cee35a6be8d0c68e09e"
+
+# Use the date of the above commit as the package version. Update this when
+# SRCREV is changed.
+PV = "20200624"
```

Comment:
Please abstract common bits between userland recipe and this recipe into an inc file e.g. all above this line

Reply:
It already **is** a .inc file you are referring to. Or maybe I'm missing what you mean.

Reply:
I think @kraj missed that this is a `inc` and not a separate `bb`.